### ISSUE TITLE
[Broken Website] icloud.com

### ISSUE THREAD

**Issue by Elaborendum** [darkreader/darkreader#6305]
# Broken Website
## Website

https://icloud.com

## How to Reproduce
  - Go to icloud.com
  - Create a new file in Pages, Keynote or Numbers
  - See that the document/presentation/spreadsheet has inconsistent coloring.

## Expected behavior

Follow what happens in Google Docs and Microsoft Office Online.

## Actual behavior

Parts of the document/presentation/spreadsheet stay their original colors, some don't, making some text unreadable.

## Screenshots

Pages:
![imagen](https://user-images.githubusercontent.com/71190696/126099392-ea53028e-5053-4576-a484-00679228d17b.png)
Keynote:
![imagen](https://user-images.githubusercontent.com/71190696/126099396-9c582a9b-88d6-4edf-aeae-24697861404c.png)
Numbers:
![imagen](https://user-images.githubusercontent.com/71190696/126099399-3aa03b88-86ad-4201-9045-0c7d08627b4c.png)

## System Information

- OS: Windows 10
- Browser: Firefox 90.0
- Dark Reader Version: 4.9.34

**Comment by Gusted** [comment]
Going to pass this one as all attempts made to create an account are being "soft-blocked" and thus not allowing me to create such files.

**Comment by arche-dev** [comment]
I have an account but I don't have my phone with me rn (2fa) :c

**Comment by yazan1107** [comment]
yes please fix this also the reminders/notes app have the same issue.

**Comment by direrFlood** [comment]
I'm having the same issue with iCloud Notes and iCloud Remidner And iCloud beta website.
